README.md

```diff
@@ -25,6 +27,20 @@ runs:
 
 These setup steps are run after the repository has been checked out and after Go has been installed, but before any tests or checks are run.
 
+### Configuration
+
+`go-check` contains an optional step that checks that running `go generate` doesn't change any files.
+This is useful to make sure that the generated code stays in sync, however, it requires code generators to be deterministic.
```

Comment:
nit: I wouldn't say "however", but rather something a bit more positive like "please note that"